packages/unmock-cli/src/index.ts

```diff
@@ -1,7 +1,47 @@
 import * as program from "commander";
+import { init, curl, list, open } from "./commands/";
 
 export default () => {
+  const collect = (val: string, memo: string[]) => {
+    memo.push(val);
+    return memo;
+  };
+
   program.usage("unmock <command>");
-  program.command("init");
+
+  program
```

Comment:
Not for this PR, but I have very much enjoyed using [oclif](https://github.com/oclif/oclif) from Heroku, it makes it easy to create modular CLIs with easy testability.

Reply:
I'll note that.